loadtest/config.go

```diff
@@ -135,6 +135,8 @@ type UsersConfiguration struct {
 	MaxActiveUsers int `default:"2000" validate:"range:(0,]"`
 	// The average number of sessions per user.
 	AvgSessionsPerUser int `default:"1" validate:"range:[1,]"`
+	// The percentage of users generated that will be system admins
+	PercentOfUsersAreAdmin float64 `default:"0.02" validate:"range:[0,1]"`
```

Comment:
A value of `0.02` seems a bit high as it will result in logging 40 admins if running with defaults. I think running one or two would be more reasonable.

We'll need to update a couple of related things as well:

- https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/config/config.md#usersconfiguration
- https://github.com/mattermost/mattermost-load-test-ng/blob/9fe059e92a159617b2787004c3e47ae6633e4569/config/config.sample.json#L48-L52

Reply:
So we have a bit of a balancing act to do here, because the frequency in which the report is run is also the absolute lowest, so I don't know how often that would be run vs. the number of admins. If the plan here is a load test though, then maybe more than 1 or 2 admins is warranted.

I would expect we go for worst-case scenario here, so I think more than 1 or 2 is better, but maybe also less than 40. @agnivade @agarciamontoro Thoughts?

I'll add the sample config.

Reply:
Fair enough, generally though I'd rather bump the frequency of the action than skewing the simulation with more users. Maybe we could refine this by gathering some data on percentage of active admins from our reference customer. Something to think about @agarciamontoro.

Reply:
Will review the PR again tomorrow, but asked about this percentage [here](https://community.mattermost.com/private-core/pl/b39wqaat4bg1p8euzcuzo5r3ia):
Community: 0.09%
Reference customer: 0.05%

Reply:
Just to confirm, in the config we use the [0,1] range for percentage, so I'd expect those values to be 100 times lower when set.